features/user_registration.feature

```diff
@@ -0,0 +1,14 @@
+Feature: As a visitor
+In order to checkout
+I would like to create an account
+
+
+Scenario: I create an account
```

Comment:
Okay, so @casperjureskog should probably articulate his thoughts a little better but what we discussed is that we should try to focus on happy paths for the MVP and disregard the corner cases (at least when we need to implement those solutions).  I think that this is generally a good idea. In this case though, since Devise is giving us a lot of functionality out-of--the-box, we should probably add tests for sad paths.